### PR TITLE
Environment cleanup round two

### PR DESCRIPTION
**Pull request by sajith**
Issue is #165.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,38 +18,39 @@ services:
...
     image: sdx-controller
     container_name: sdx-controller
+    depends_on:
+      - sdx-controller-mongodb-service
     tty: true
     build: ./
     ports:
       - 8080:8080
     environment:
-      # Use sdx-controller-mongodb service specified above.
```
Comment (congwang09) [inline docker-compose.yml:30]: Are the env vars being moved to .env file?

**Review by congwang09 — COMMENTED** [review]
Just quick question below.

File: docker-compose.yml (hunk 1 above)
Comment (sajith) [reply docker-compose.yml:30]: Yes. However MongoDB connection would be using Docker Compose networking, so sdx-controller will be connecting to the `sdx-controller-mongodb` container.  No need of making that an environment variable.

Comment (congwang09) [reply docker-compose.yml:30]: I mean, remove all the `- XX='xxx'`, and replace with something like:
```
environment:
- .env
```

Comment (sajith) [reply docker-compose.yml:30]: To be clear, I am not too opposed to just setting `env_file`. That perhaps is simpler, and the other details are not worth the worry? It probably is just a matter of opinion?

I am not a Docker expert, so my opinion here isn't worth much. :-)

**Review by congwang09 — APPROVED** [review]

**Pull request merged**